packages/synapse-react-client/src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -16,7 +16,7 @@ dayjs.extend(utc)
 dayjs.extend(timezone)
 dayjs.extend(advancedFormat)
 
-export type DateTimePickerProps = MuiDateTimePickerProps<string | Dayjs>
+export type DateTimePickerProps = MuiDateTimePickerProps<Dayjs>
```

Comment:
Change our DateTimePicker to only use Dayjs. The point is that how we manage datetimes can vary across the different services, so we should force whoever is using the component to deal with the complexity of parsing and returning the correct format.